ontology_api/role/role_model.py

```diff
@@ -7,6 +7,11 @@ class RoleModelIn(BaseModel):
     instance_name: Optional[str]
     value: Union[str, int, None]
 
+class RolesDeletedOut(BaseModel):
+    model_id: Optional[int]
```

Comment:
Remove model_id. It's not useful here. We've used this id only in ModelOut. We should hold on some convention. If we want to resend model_id. We should do it everytime. And it would require big change in all ontology_api